tests/test_cases/issue_348/issue_348.py

```diff
@@ -1,27 +1,23 @@
 import cocotb
 from cocotb.log import SimLog
 from cocotb.triggers import Timer, Edge, RisingEdge, FallingEdge
-from cocotb.result import TestFailure
 
 
-@cocotb.coroutine
-def clock_gen(signal, num):
+async def clock_gen(signal, num):
     for x in range(num):
         signal <= 0
-        yield Timer(500)
+        await Timer(500)
         signal <= 1
-        yield Timer(500)
+        await Timer(500)
 
 
-@cocotb.coroutine
-def signal_mon(signal, idx, edge):
-    log = SimLog("cocotb.signal_mon.%d.%s" % (idx, signal._name))
-    value = signal.value
-
+async def signal_mon(signal, idx, edge):
+    _ = SimLog("cocotb.signal_mon.%d.%s" % (idx, signal._name))
+    _ = signal.value
```

Comment:
Why the `_`? Should we just delete these lines?

Reply:
I had intended to come back to this, but forgot. Yes, it's dead code. I'll remove.
```suggestion
```